pkg/virt-controller/watch/vm_test.go

```diff
@@ -1178,6 +1187,334 @@ var _ = Describe("VirtualMachine", func() {
 				})
 			})
 		})
+
+		Context("VM printableStatus", func() {
+
```

Comment:
It seems that many of the different tests below are very similar. Maybe you can create a more general table function that will allow you to put most of these tests into a single table ? 
For example "should set a Stopping status when VMI has a deletion timestamp set"   and ""should set a Terminating status when VM has a deletion timestamp set"  seems almost identical ...

Reply:
There are subtle differences between these tests so most of them ended up in `It()`s of their own. I tried to gather to `table.DescribeTable` where it made sense. Nevertheless I'll go through it again and see if I can gather up a few more.

Reply:
Having another look, I think that merging more of these tests to parameterized tables would yield test code with too many conditionals and/or obscure testcase parameters. I prefer to leave as is, despite the inevitable duplication.